README.md

```diff
@@ -41,3 +41,203 @@ Package can then be placed directly under `./charts/<centos|redhat>/<chart-name>
+
+```commandline
+$ helm package <path_to_helm_chart>
+```
+
+E.g. Using postgresql-persistent Helm chart:
+
+```commandline
+$ helm package charts/redhat/postgresql-persistent/0.0.1
+Successfully packaged chart and saved it to: <FULL_PATH_TO_CWD>/helm-charts/postgresql-persistent-0.0.1.tgz
+```
+
+#### Deploy Helm chart to OpenShift
+
+You can deploy the Helm chart to the OpenShift cluster using command:
+
+```commandline
+$ helm install <helm_chart_name> <tar_ball_path>
+```
+
+E.g Using postgresql-persistent Helm chart:
+
+```commandline
+$ helm install postgresql-imagestreams postgresql-imagestreams-0.0.1.tgz
+NAME: postgresql-imagestreams001
+LAST DEPLOYED: Tue Apr  4 10:29:33 2023
+NAMESPACE: pgsql-13
+STATUS: deployed
+REVISION: 1
+TEST SUITE: None
+```
+
+To check if Helm chart is really deployed, run command:
+
+```bash
+helm list
+```
+
+Examples of the `helm test` output:
+
+```commandline
+$ helm list
+NAME                      	NAMESPACE	REVISION	UPDATED                              	STATUS  	CHART                        	APP VERSION
+postgresql-imagestreams  	pgsql-13 	1       	2023-04-04 10:29:33.878674 +0200 CEST	deployed	postgresql-imagestreams-0.0.1	0.0.1
+postgresql-persistent     	pgsql-13 	1       	2023-03-28 16:59:56.498783 +0200 CEST	deployed	postgresql-persistent-v0.0.1
+```
+
+### How to verify Helm chart
+
+As soon as a Helm chart is ready and tested let's verify it. The first we have to create a package by command:
+
+```commandline
+$ helm test <helm_chart_name> --logs
+```
+
+The output should be like:
+
+```commandline
+$ helm test postgresql-persistent --logs
+NAME: postgresql-persistent
+LAST DEPLOYED: Tue Mar 28 16:59:56 2023
+NAMESPACE: pgsql-13
+STATUS: deployed
+REVISION: 1
+TEST SUITE:     postgresql-persistent-connection-test
+Last Started:   Tue Apr  4 10:31:44 2023
+Last Completed: Tue Apr  4 10:31:52 2023
+Phase:          Succeeded
+
+POD LOGS: postgresql-persistent-connection-test
+postgresql-testing:5432 - accepting connections
+```
+
+
+In case the Helm chart has already been merged,
+use the following command to verify if all the required information is contained within the Helm chart.
+
+```commandline
+docker run --rm -i -e KUBECONFIG=/.kube/config -v "${HOME}/.kube":/.kube \
+          "quay.io/redhat-certification/chart-verifier:latest" \
+          verify \
+          https://sclorg.github.io/helm-charts/<helm-chart-name>.tgz
+```
+
+Use the following command to verify if all the required information for local testing is contained within the Helm chart.
+
+```commandline
+docker run --rm -i -e KUBECONFIG=/.kube/config -v "${HOME}/.kube":/.kube \
+          -v "<full_path_to_this_dir>":/helm-charts
+          "quay.io/redhat-certification/chart-verifier:latest" \
+          verify \
+          /helm-charts/<helm-chart-name>.tgz
+```
+
+The output from commands mentioned above could be:
```

Comment:
s/be/look like

Reply:
https://github.com/sclorg/helm-charts/pull/16/commits/d5b36a15a48a6d66c812118bbe0450ad6c32f4fd